app/Http/Controllers/GameController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class GameController extends Controller
+{
+    public function gameList(?int $id = null)
+    {
+        return view('game-list', [
+            'games'=>$this->getGames(),
+        ]);
+    }
+
+
+    private function getGames(): array
+    {
+        return [
+            new Game(1, 'Monopoly', 5.0, '8+'),
+            new Game(2, 'Risk', 6.0, '10+'),
+            new Game(3, 'DND', 9.0, '12+'),
+        ];
+    }
+
+    public function getGameById(int $id): Game
```

Comment:
Where is this function used? Is it supposed to be an action? I don't see it in the routes file, and it doesn't return a game-info view as mentioned in #3